projects/dashboards/data-atlas/src/app/main/main.component.html

```diff
@@ -10,7 +10,20 @@
                     </mat-panel-description>
                 </mat-expansion-panel-header>
 
-                <geoengine-accordion-entry [collection]="collection.id" [icon]="icon(collection.name)"></geoengine-accordion-entry>
+                <mat-tab-group dynamicHeight>
+                    <mat-tab label="Raster" *ngIf="collection.raster">
+                        <geoengine-accordion-entry
+                            [collection]="collection.raster.id"
+                            [icon]="icon(collection.name)"
+                        ></geoengine-accordion-entry>
+                    </mat-tab>
+                    <mat-tab label="Vector" *ngIf="collection.vector">
```

Comment:
it is really strange that it does not show when there is no raster in the matching folder. Is it possible to change that?

Reply:
[7770ee7](https://github.com/geo-engine/geoengine-ui/pull/611/commits/7770ee7cd15f5059818e6175ea48f9016eefa7b5)